platform/linux-dpdk/Makefile.am

```diff
@@ -246,7 +246,6 @@ __LIB__libodp_dpdk_la_SOURCES = \
 			   ../linux-generic/pktio/loop.c \
 			   ../linux-generic/odp_pkt_queue.c \
 			   pool/dpdk.c \
-			   ../linux-generic/odp_queue.c \
```

Comment:
You may need to add the actual queue implementation... :
../linux-generic/queue/generic.c \
and subsystem...
../linux-generic/queue/subsystem.c \